lib/identity_tijuana.rb

```diff
@@ -74,7 +74,7 @@ def self.fetch_user_updates_impl(sync_id)
     started_at = DateTime.now
     last_updated_at = get_redis_date('tijuana:users:last_updated_at')
     last_id = (Sidekiq.redis { |r| r.get 'tijuana:users:last_id' } || 0).to_i
-    users_dependent_data_cutoff = DateTime.now
+    users_dependent_data_cutoff =  DateTime.now
```

Comment:
needs to go 🔙

Reply:
aww... but there is soo much space :) ... 🎸